lib/Service/TemplateFieldService.php

```diff
@@ -44,6 +50,13 @@ public function extractFields(Node|int $file) {
 				throw new NotFoundException();
 			}
 
+			$cacheName = $file->getId() . "/" . $file->getEtag();
+			$cachedResponse = $this->cache->get($cacheName);
+
+			if ($cachedResponse) {
```

Comment:
```suggestion
			if ($cachedResponse !== null) {
```

Otherwise an empty array would also evaluate to false and not use the cached result.